src/components/utils/blob.js

```diff
@@ -41,13 +41,14 @@ const base64toBlob = (image) => {
   return new Blob([base64Data.integerArray], {type: base64Data.mimeString})
 }
 
-export const canvasToBlob = (canvas, callback) => {
+export const canvasToBlob = (canvas, callback, imageQuality) => {
+  const screenshotFormat = 'image/png'
   if (!HTMLCanvasElement.prototype.toBlob) {
     // Handle browsers that do not support canvas.toBlob() like Edge
     const dataUrlImg = canvas.toDataURL()
-    return callback(base64toBlob(dataUrlImg))
+    return callback(base64toBlob(dataUrlImg, screenshotFormat, imageQuality))
   }
-  return canvas.toBlob(callback, "image/png")
+  return canvas.toBlob(callback, screenshotFormat, imageQuality)
```

Comment:
What happens if imageQuality is null?

Reply:
I think it is ignored. Could set a default value to be safe. 

According to the MDN documentation for `canvas.onBlob()` method:
> If this argument is anything else, the default values 0.92 and 0.80 are used for image/jpeg and image/webp respectively. Other arguments are ignored.

More details [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob)